src/renderers/dom/shared/ReactDOMComponent.js

```diff
@@ -263,9 +263,11 @@ function assertValidProps(component, props) {
   }
   invariant(
     props.style == null || typeof props.style === 'object',
-    'The `style` prop expects a mapping from style properties to values, ' +
+    'A dom node has incorrect inline style syntax.' +
```

Comment:
Please just leave the beginning of this message as it was. We really just need the addendum.